{
  "nbformat": 4,
  "nbformat_minor": 0,
  "metadata": {
    "colab": {
      "name": "pyserini_covid19_paragraph.ipynb",
      "provenance": [],
      "collapsed_sections": [],
      "include_colab_link": true
    },
    "kernelspec": {
      "name": "python3",
      "display_name": "Python 3"
    }
  },
  "cells": [
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "view-in-github",
        "colab_type": "text"
      },
      "source": [
        "<a href=\"https://colab.research.google.com/github/castorini/anserini-notebooks/blob/master/pyserini_covid19_paragraph.ipynb\" target=\"_parent\"><img src=\"https://colab.research.google.com/assets/colab-badge.svg\" alt=\"Open In Colab\"/></a>"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "vUpmSFflwzR1",
        "colab_type": "text"
      },
      "source": [
        "# Pyserini Demo on COVID-19 Dataset (Paragraph Index)\n"
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "0RtnmBN6A-Oq",
        "colab_type": "text"
      },
      "source": [
        "This notebook provides a demo on how to get started in searching the [COVID-19 Open Research Dataset](https://pages.semanticscholar.org/coronavirus-research) (release of 2020/04/03) from AI2.\n",
        "Here, we'll be working with the paragraph index.\n",
        "We have [another notebook](https://github.com/castorini/anserini-notebooks/blob/master/pyserini_covid19_default.ipynb) for working with the simpler title + abstract index."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "f2vNd7bpJlDZ",
        "colab_type": "text"
      },
      "source": [
        "First, install Python dependencies"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "E_lt0-pXJia0",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "%%capture\n",
        "!pip install pyserini==0.8.1.0\n",
        "\n",
        "import json\n",
        "import os\n",
        "os.environ[\"JAVA_HOME\"] = \"/usr/lib/jvm/java-11-openjdk-amd64\""
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "xbXmstbGypef",
        "colab_type": "text"
      },
      "source": [
        "Let's grab the pre-built index:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "JT_OJKftdqGP",
        "colab_type": "code",
        "colab": {}
      },
      "source": [
        "%%capture\n",
        "!wget https://www.dropbox.com/s/rfzxrrstwlck4wh/lucene-index-covid-paragraph-2020-04-03.tar.gz\n",
        "!tar xvfz lucene-index-covid-paragraph-2020-04-03.tar.gz"
      ],
      "execution_count": 0,
      "outputs": []
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "EwVSvHBz9RMM",
        "colab_type": "text"
      },
      "source": [
        "Sanity check of index size (should be 5.3G):"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "KVXWA6WS0aqJ",
        "colab_type": "code",
        "outputId": "316b3f3b-bd2d-45f4-fed0-9c6d0a3555be",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 34
        }
      },
      "source": [
        "!du -h lucene-index-covid-paragraph-2020-04-03"
      ],
      "execution_count": 3,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "5.3G\tlucene-index-covid-paragraph-2020-04-03\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "WW-vWrP41fAz",
        "colab_type": "text"
      },
      "source": [
        "Now, a bit of explanation of how the index is organized.\n",
        "For each source article, we create a paragraph-level index as follows, for a hypothetical article with id `docid`, in the index there'll be:\n",
        "\n",
        "+ `docid`: title + abstract\n",
        "+ `docid.00001`: title + abstract + 1st paragraph\n",
        "+ `docid.00002`: title + abstract + 2nd paragraph\n",
        "+ `docid.00003`: title + abstract + 3rd paragraph\n",
        "+ ...\n",
        "\n",
        "That is, each article is chopped up into individual paragraphs.\n",
        "Each paragraph is indexed as a \"document\" (with the title and abstract). \n",
        "The suffix of the `docid`, `.XXXXX` identifies which paragraph is being indexed (numbered sequentially)."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "NkD0kKxW9mHP",
        "colab_type": "text"
      },
      "source": [
        "You can use `pysearch` to search over an index. Here's the basic usage:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "yFZlcqEX0t1f",
        "colab_type": "code",
        "outputId": "68389c57-6cef-437f-f7bc-3485bce0c930",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 207
        }
      },
      "source": [
        "from pyserini.search import pysearch\n",
        "\n",
        "searcher = pysearch.SimpleSearcher('lucene-index-covid-paragraph-2020-04-03/')\n",
        "hits = searcher.search('nsp1 synthesis degradation', 10)\n",
        "\n",
        "# Prints the first 10 hits\n",
        "for i in range(0, 10):\n",
        "    print(f'{i+1:2} {hits[i].docid:14} {hits[i].score:.5f} {hits[i].lucene_document.get(\"title\")} {hits[i].lucene_document.get(\"doi\")}')"
      ],
      "execution_count": 4,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            " 1 o328y8ax.00004 10.76930 Modulation of type I interferon induction by porcine reproductive and respiratory syndrome virus and degradation of CREB-binding protein by non-structural protein 1 in MARC-145 and HeLa cells 10.1016/j.virol.2010.03.039\n",
            " 2 06z7p7rc       10.68730 Severe Acute Respiratory Syndrome Coronavirus nsp1 Suppresses Host Gene Expression, Including That of Type I Interferon, in Infected Cells 10.1128/jvi.02472-07\n",
            " 3 mtj46j82.00029 10.51600 MERS coronavirus nsp1 participates in an efficient propagation through a specific interaction with viral RNA 10.1016/j.virol.2017.08.026\n",
            " 4 42saxb98.00002 10.50550 A novel two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein 10.1038/nsmb.1680\n",
            " 5 42saxb98.00001 10.50230 A novel two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein 10.1038/nsmb.1680\n",
            " 6 vj3wk150.00035 10.49290 Regulation of Stress Responses and Translational Control by Coronavirus 10.3390/v8070184\n",
            " 7 sq9hh50d.00005 10.43700 Unique SARS-CoV protein nsp1: bioinformatics, biochemistry and potential effects on virulence 10.1016/j.tim.2006.12.005\n",
            " 8 42saxb98.00020 10.43350 A novel two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein 10.1038/nsmb.1680\n",
            " 9 pdfs6ojs.00027 10.41460 Coronavirus nonstructural protein 1: Common and distinct functions in the regulation of host and viral gene expression 10.1016/j.virusres.2014.11.019\n",
            "10 fe1yj05b       10.38410 Severe acute respiratory syndrome coronavirus nsp1 protein suppresses host gene expression by promoting host mRNA degradation 10.1073/pnas.0603144103\n"
          ],
          "name": "stdout"
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "N6fk6kKQ2t-s",
        "colab_type": "text"
      },
      "source": [
        "From the hits array, use `.lucene_document` to access the underlying indexed Lucene `Document`, and from there, call `.get(field)` to fetch specific fields, like \"title\", \"doc\", etc.\n",
        "The complete list of available fields is [here](https://github.com/castorini/anserini/blob/master/src/main/java/io/anserini/index/generator/CovidGenerator.java#L46).\n",
        "\n",
        "Note that we retrieve multiple paragraphs from the same article \"A novel two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein\" (hits #4 and #5). Note that this actually a good thing, because a downstream module can do evidence integration.\n",
        "\n",
        "Considering hit #4 (`42saxb98.00002`) and hit #5 (`42saxb98.00001`), use `.contents` of the hit to see exactly what was indexed.\n",
        "\n",
        "For hit #4:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "-EqwvCMU2-Ft",
        "colab_type": "code",
        "outputId": "6cf10b0b-a8e1-4d28-817c-6face81d11c7",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 88
        }
      },
      "source": [
        "hits[3].contents.split('\\n')"
      ],
      "execution_count": 5,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['A novel two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein',\n",
              " \"Severe acute respiratory syndrome coronavirus nsp1 protein suppresses host gene expression, including type I interferon production, by promoting host mRNA degradation and inhibiting host translation, in infected cells. We present evidence that nsp1 uses a novel, two-pronged strategy to inhibit host translation/gene expression. Nsp1 bound to the 40S ribosomal subunit and inactivated the translational activity of the 40S subunits. Furthermore, the nsp1-40S ribosome complex induced the modification of the 5'-region of capped mRNA template and rendered the template RNA translationally incompetent. Nsp1 also induced RNA cleavage in templates carrying the internal ribosome entry site (IRES) from encephalomyocarditis virus, but not in those carrying IRESs from hepatitis C and cricket paralysis viruses, demonstrating that the nsp1-induced RNA modification was template-dependent. We speculate that the mRNAs that underwent the nsp1-mediated modification are marked for rapid turnover by the host RNA degradation machinery.\",\n",
              " 'We tested whether nsp1 protein could suppress translation in rabbit reticulocyte lysate (RRL). We expressed the full-length, wild-type nsp1 protein and its mutant form (nsp1-mt) carrying K164A and H165A mutations as glutathione S-transferase (GST)-tagged fusion proteins in E.coli followed by the removal of the GST tag to generate recombinant nsp1 and nsp1-mt proteins, respectively; nsp1-mt neither suppresses host translation nor promotes host mRNA degradation in expressing cells and infected cells10. After incubation of the nsp1 protein with RRL at 4Â°C for 30 min, we added different concentrations of capped and polyadenylated Renilla luciferase mRNA transcripts (rLuc RNA) to the mixture, and incubated the samples in the presence of [ 35 S]methionine for 30 min. In control samples in which rLuc RNA was incubated with GST and nsp1-mt proteins, rLuc activity and labeled rLuc protein levels increased with rising mRNA concentrations. In contrast, the rLuc activity and labeled rLuc protein levels in the nsp1-containing sample was substantially lower than in the control samples; the rLuc activity and the radiolabeled rLuc protein levels in the presence of nsp1 were about 6-8% of the levels observed with GST or nsp1-mt ( Fig.  1a and b) , which clearly demonstrated that nsp1 efficiently inhibited the rLuc protein synthesis from capped rLuc RNA in RRL.']"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 5
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "OfL1RYGE31fH",
        "colab_type": "text"
      },
      "source": [
        "For hit #5:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "HcvgFupQ3xPJ",
        "colab_type": "code",
        "outputId": "7c033937-3256-43db-d4a9-0b2181dc31bf",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 88
        }
      },
      "source": [
        "hits[4].contents.split('\\n')"
      ],
      "execution_count": 6,
      "outputs": [
        {
          "output_type": "execute_result",
          "data": {
            "text/plain": [
              "['A novel two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein',\n",
              " \"Severe acute respiratory syndrome coronavirus nsp1 protein suppresses host gene expression, including type I interferon production, by promoting host mRNA degradation and inhibiting host translation, in infected cells. We present evidence that nsp1 uses a novel, two-pronged strategy to inhibit host translation/gene expression. Nsp1 bound to the 40S ribosomal subunit and inactivated the translational activity of the 40S subunits. Furthermore, the nsp1-40S ribosome complex induced the modification of the 5'-region of capped mRNA template and rendered the template RNA translationally incompetent. Nsp1 also induced RNA cleavage in templates carrying the internal ribosome entry site (IRES) from encephalomyocarditis virus, but not in those carrying IRESs from hepatitis C and cricket paralysis viruses, demonstrating that the nsp1-induced RNA modification was template-dependent. We speculate that the mRNAs that underwent the nsp1-mediated modification are marked for rapid turnover by the host RNA degradation machinery.\",\n",
              " \"Severe acute respiratory syndrome (SARS) coronavirus (SCoV), the causative agent of a newly emerged disease, SARS, is an enveloped virus that contains a single-stranded, positive-sense RNA genome of about 29.7 kb.1 Upon infection, the genome expression begins with the translation of gene 1, which constitutes the 5'-end two-thirds of the viral genome, to produce two large precursor polyproteins2,3 that are proteolytically processed by 2 virally encoded proteinases to generate 16 mature proteins, namely nsp1 to nsp162. While Users may view, print, copy, download and text and data-mine the content in such documents, for the purposes of academic research, subject always to the full Conditions of use: http://www.nature.com/authors/editorial_policies/license.html#terms most of these gene 1 proteins are essential for viral RNA synthesis, some of them appear to have other biological functions. [4] [5] [6] [7] The most N-terminal gene 1 protein, nsp1, has unique biological functions; expressed SCoV nsp1 induces host mRNA degradation and suppresses host translation8. The expressed nsp1 suppresses the host antiviral signaling pathways as well9. Furthermore, nsp1 suppresses host gene expression, including type I interferon (IFN) production, by promoting host mRNA degradation and host translation suppression in infected cells10. The nsp1 of a closely related mouse hepatitis virus also suppresses host gene expression and is a viral virulence factor11. These data suggest that the SCoV nsp1-mediated suppression of host genes plays an important role in the pathogenesis of SARS. Accordingly, a delineation of the mechanisms of the nsp1-induced suppression of host gene expression is considerably important for gaining insight into SCoV pathogenesis at the molecular level. We designed the present study to uncover the mechanism of nsp1-induced suppression of host gene expression primarily by using an in vitro system. Our data revealed that nsp1 utilizes a novel, two-pronged strategy to inhibit host translation/gene expression.\"]"
            ]
          },
          "metadata": {
            "tags": []
          },
          "execution_count": 6
        }
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "qrwpdqXL3_5j",
        "colab_type": "text"
      },
      "source": [
        "The first two lines contain the title and abstract, respectively, and they are exactly the same for both, since they're from the same article."
      ]
    },
    {
      "cell_type": "markdown",
      "metadata": {
        "id": "iiXaE-TDH7uc",
        "colab_type": "text"
      },
      "source": [
        "To access the full text, we need to fetch the \"base\" document, which is `42saxb98` (without the `.XXXXX` suffix).\n",
        "This is to avoid wasting space by repeatedly storing the full text.\n",
        "\n",
        "We can use the `searcher` to fetch the document, and then fetch the underlying raw article JSON, as follows:"
      ]
    },
    {
      "cell_type": "code",
      "metadata": {
        "id": "WAYzpshzL7St",
        "colab_type": "code",
        "outputId": "9ed7fa5d-e158-4bb3-fe42-636172d78fb9",
        "colab": {
          "base_uri": "https://localhost:8080/",
          "height": 1000
        }
      },
      "source": [
        "article = json.loads(searcher.doc('42saxb98').lucene_document().get('raw'))\n",
        "print(json.dumps(article, indent=4))"
      ],
      "execution_count": 7,
      "outputs": [
        {
          "output_type": "stream",
          "text": [
            "{\n",
            "    \"paper_id\": \"8ceb037798bd3fa6941261d1b888fe0cb79f2850\",\n",
            "    \"metadata\": {\n",
            "        \"title\": \"A novel two-pronged strategy to suppress host protein synthesis by SARS coronavirus Nsp1 protein\",\n",
            "        \"authors\": [\n",
            "            {\n",
            "                \"first\": \"Wataru\",\n",
            "                \"middle\": [],\n",
            "                \"last\": \"Kamitani\",\n",
            "                \"suffix\": \"\",\n",
            "                \"affiliation\": {\n",
            "                    \"laboratory\": \"\",\n",
            "                    \"institution\": \"The University of Texas Medical Branch at Galveston\",\n",
            "                    \"location\": {\n",
            "                        \"postCode\": \"77555-1019\",\n",
            "                        \"settlement\": \"Galveston\",\n",
            "                        \"region\": \"Texas\"\n",
            "                    }\n",
            "                },\n",
            "                \"email\": \"\"\n",
            "            },\n",
            "            {\n",
            "                \"first\": \"Cheng\",\n",
            "                \"middle\": [],\n",
            "                \"last\": \"Huang\",\n",
            "                \"suffix\": \"\",\n",
            "                \"affiliation\": {\n",
            "                    \"laboratory\": \"\",\n",
            "                    \"institution\": \"The University of Texas Medical Branch at Galveston\",\n",
            "                    \"location\": {\n",
            "                        \"postCode\": \"77555-1019\",\n",
            "                        \"settlement\": \"Galveston\",\n",
            "                        \"region\": \"Texas\"\n",
            "                    }\n",
            "                },\n",
            "                \"email\": \"\"\n",
            "            },\n",
            "            {\n",
            "                \"first\": \"Krishna\",\n",
            "                \"middle\": [],\n",
            "                \"last\": \"Narayanan\",\n",
            "                \"suffix\": \"\",\n",
            "                \"affiliation\": {\n",
            "                    \"laboratory\": \"\",\n",
            "                    \"institution\": \"The University of Texas Medical Branch at Galveston\",\n",
            "                    \"location\": {\n",
            "                        \"postCode\": \"77555-1019\",\n",
            "                        \"settlement\": \"Galveston\",\n",
            "                        \"region\": \"Texas\"\n",
            "                    }\n",
            "                },\n",
            "                \"email\": \"\"\n",
            "            },\n",
            "            {\n",
            "                \"first\": \"Kumari\",\n",
            "                \"middle\": [\n",
            "                    \"G\"\n",
            "                ],\n",
            "                \"last\": \"Lokugamage\",\n",
            "                \"suffix\": \"\",\n",
            "                \"affiliation\": {\n",
            "                    \"laboratory\": \"\",\n",
            "                    \"institution\": \"The University of Texas Medical Branch at Galveston\",\n",
            "                    \"location\": {\n",
            "                        \"postCode\": \"77555-1019\",\n",
            "                        \"settlement\": \"Galveston\",\n",
            "                        \"region\": \"Texas\"\n",
            "                    }\n",
            "                },\n",
            "                \"email\": \"\"\n",
            "            },\n",
            "            {\n",
            "                \"first\": \"Shinji\",\n",
            "                \"middle\": [],\n",
            "                \"last\": \"Makino\",\n",
            "                \"suffix\": \"\",\n",
            "                \"affiliation\": {\n",
            "                    \"laboratory\": \"\",\n",
            "                    \"institution\": \"The University of Texas Medical Branch at Galveston\",\n",
            "                    \"location\": {\n",
            "                        \"postCode\": \"77555-1019\",\n",
            "                        \"settlement\": \"Galveston\",\n",
            "                        \"region\": \"Texas\"\n",
            "                    }\n",
            "                },\n",
            "                \"email\": \"\"\n",
            "            }\n",
            "        ]\n",
            "    },\n",
            "    \"abstract\": [\n",
            "        {\n",
            "            \"text\": \"Severe acute respiratory syndrome coronavirus nsp1 protein suppresses host gene expression, including type I interferon production, by promoting host mRNA degradation and inhibiting host translation, in infected cells. We present evidence that nsp1 uses a novel, two-pronged strategy to inhibit host translation/gene expression. Nsp1 bound to the 40S ribosomal subunit and inactivated the translational activity of the 40S subunits. Furthermore, the nsp1-40S ribosome complex induced the modification of the 5'-region of capped mRNA template and rendered the template RNA translationally incompetent. Nsp1 also induced RNA cleavage in templates carrying the internal ribosome entry site (IRES) from encephalomyocarditis virus, but not in those carrying IRESs from hepatitis C and cricket paralysis viruses, demonstrating that the nsp1-induced RNA modification was template-dependent. We speculate that the mRNAs that underwent the nsp1mediated modification are marked for rapid turnover by the host RNA degradation machinery.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Abstract\"\n",
            "        }\n",
            "    ],\n",
            "    \"body_text\": [\n",
            "        {\n",
            "            \"text\": \"Severe acute respiratory syndrome (SARS) coronavirus (SCoV), the causative agent of a newly emerged disease, SARS, is an enveloped virus that contains a single-stranded, positive-sense RNA genome of about 29.7 kb.1 Upon infection, the genome expression begins with the translation of gene 1, which constitutes the 5'-end two-thirds of the viral genome, to produce two large precursor polyproteins2,3 that are proteolytically processed by 2 virally encoded proteinases to generate 16 mature proteins, namely nsp1 to nsp162. While Users may view, print, copy, download and text and data-mine the content in such documents, for the purposes of academic research, subject always to the full Conditions of use: http://www.nature.com/authors/editorial_policies/license.html#terms most of these gene 1 proteins are essential for viral RNA synthesis, some of them appear to have other biological functions. [4] [5] [6] [7] The most N-terminal gene 1 protein, nsp1, has unique biological functions; expressed SCoV nsp1 induces host mRNA degradation and suppresses host translation8. The expressed nsp1 suppresses the host antiviral signaling pathways as well9. Furthermore, nsp1 suppresses host gene expression, including type I interferon (IFN) production, by promoting host mRNA degradation and host translation suppression in infected cells10. The nsp1 of a closely related mouse hepatitis virus also suppresses host gene expression and is a viral virulence factor11. These data suggest that the SCoV nsp1-mediated suppression of host genes plays an important role in the pathogenesis of SARS. Accordingly, a delineation of the mechanisms of the nsp1-induced suppression of host gene expression is considerably important for gaining insight into SCoV pathogenesis at the molecular level. We designed the present study to uncover the mechanism of nsp1-induced suppression of host gene expression primarily by using an in vitro system. Our data revealed that nsp1 utilizes a novel, two-pronged strategy to inhibit host translation/gene expression.\",\n",
            "            \"cite_spans\": [\n",
            "                {\n",
            "                    \"start\": 899,\n",
            "                    \"end\": 902,\n",
            "                    \"text\": \"[4]\",\n",
            "                    \"ref_id\": \"BIBREF3\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 903,\n",
            "                    \"end\": 906,\n",
            "                    \"text\": \"[5]\",\n",
            "                    \"ref_id\": \"BIBREF4\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 907,\n",
            "                    \"end\": 910,\n",
            "                    \"text\": \"[6]\",\n",
            "                    \"ref_id\": \"BIBREF5\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 911,\n",
            "                    \"end\": 914,\n",
            "                    \"text\": \"[7]\",\n",
            "                    \"ref_id\": \"BIBREF6\"\n",
            "                }\n",
            "            ],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Introduction\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We tested whether nsp1 protein could suppress translation in rabbit reticulocyte lysate (RRL). We expressed the full-length, wild-type nsp1 protein and its mutant form (nsp1-mt) carrying K164A and H165A mutations as glutathione S-transferase (GST)-tagged fusion proteins in E.coli followed by the removal of the GST tag to generate recombinant nsp1 and nsp1-mt proteins, respectively; nsp1-mt neither suppresses host translation nor promotes host mRNA degradation in expressing cells and infected cells10. After incubation of the nsp1 protein with RRL at 4\\u00b0C for 30 min, we added different concentrations of capped and polyadenylated Renilla luciferase mRNA transcripts (rLuc RNA) to the mixture, and incubated the samples in the presence of [ 35 S]methionine for 30 min. In control samples in which rLuc RNA was incubated with GST and nsp1-mt proteins, rLuc activity and labeled rLuc protein levels increased with rising mRNA concentrations. In contrast, the rLuc activity and labeled rLuc protein levels in the nsp1-containing sample was substantially lower than in the control samples; the rLuc activity and the radiolabeled rLuc protein levels in the presence of nsp1 were about 6-8% of the levels observed with GST or nsp1-mt ( Fig.  1a and b) , which clearly demonstrated that nsp1 efficiently inhibited the rLuc protein synthesis from capped rLuc RNA in RRL.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 1233,\n",
            "                    \"end\": 1248,\n",
            "                    \"text\": \"Fig.  1a and b)\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Nsp1 suppresses translation in vitro\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Next, we examined the effect of nsp1 on translation mediated by the internal ribosome entry sites (IRES) using in vitro-synthesized, bicistronic mRNAs, Ren-HCV-FF RNA and Ren-CrPV-FF RNA, in which expression of the upstream rLuc open reading frame (ORF) was mediated by cap-dependent translation and the translation of downstream firefly luciferase (fLuc) ORF was driven by hepatitis C virus (HCV) IRES and cricket paralysis virus (CrPV) IRES, respectively (Fig. 1c) . Nsp1 efficiently suppressed the translation of capped fLuc RNA, m 7 G-FF, as well as translation driven by both IRESs (Fig. 1d-e) . According to the current models of translation initiation, the CrPV-IRES recruits the 40S ribosomal subunit in the absence of any initiation factors12,13, which suggested to us that nsp1 could suppress the function of the 40S subunit or block translation at a post-initiation stage.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 457,\n",
            "                    \"end\": 466,\n",
            "                    \"text\": \"(Fig. 1c)\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 587,\n",
            "                    \"end\": 598,\n",
            "                    \"text\": \"(Fig. 1d-e)\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Nsp1 suppresses translation in vitro\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We tested whether nsp1 associates with the 40S ribosomal subunit and suppresses its function. We transfected 293 cells with mRNAs encoding nsp1, nsp1-mt or chloramphenicol acetyltransferase (CAT) proteins; all of these encoded proteins carried a C-terminal myc-His tag. At 8 h post transfection, we prepared the cell extracts and performed polysome profile analysis ( Fig. 2a-2c) . Expression of CAT and nsp1-mt showed similar polysome profiles, whereas nsp1 expression resulted in the accumulation of 80S monosomes, along with a significant reduction in the polysome abundance, in accordance with the observation that the expressed nsp1, but not nsp1-mt, suppresses host translation10. Western blot analysis of the resolved gradient fractions showed the co-sedimentation of the majority of nsp1, but not CAT and nsp1-mt, with the 40S ribosomal subunit (Fig. 2b) . Likewise, only nsp1 cosedimented with the 40S ribosomal subunit in RRL (Fig. 2d-2f) . To confirm the interaction of nsp1 with the 40S ribosomal subunit, we transfected 293 cells with in vitro-synthesized mRNAs encoding nsp1-myc-His fusion protein, CAT-myc-His protein or nsp1-mt-myc-His protein and subjected the cell extracts to immunoprecipitation with anti-myc antibody. The immune complexes were washed with 0.5 M KCl, a stringent high-salt condition in which the 60S ribosomal subunits and the translation initiation factors are dissociated from the 40S subunits14. Consistent with our previous report10, the expression level of nsp1 was substantially lower than that of nsp1-mt or CAT. Nevertheless, an efficient coimmunoprecipitation of 18S rRNA and S6 ribosomal protein, core components of the 40S subunit, with nsp1, but not with CAT and nsp1-mt ( Fig. 2g-2i) , suggested the tight association of nsp1 with the 40S ribosomal subunit. Similarly, co-immunoprecipitation analysis showed that nsp1 also interacted with the 40S ribosomal subunit in RRL ( Supplementary Fig. 1 ).\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 368,\n",
            "                    \"end\": 379,\n",
            "                    \"text\": \"Fig. 2a-2c)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 853,\n",
            "                    \"end\": 862,\n",
            "                    \"text\": \"(Fig. 2b)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 936,\n",
            "                    \"end\": 948,\n",
            "                    \"text\": \"(Fig. 2d-2f)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1722,\n",
            "                    \"end\": 1733,\n",
            "                    \"text\": \"Fig. 2g-2i)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1924,\n",
            "                    \"end\": 1944,\n",
            "                    \"text\": \"Supplementary Fig. 1\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Nsp1 protein interacts with the 40S ribosomal subunit\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"The finding that nsp1 inhibited CrPV-IRES-driven translation suggests that nsp1 could affect an event(s) downstream of the 43S complex formation that involves the binding of eIF2-GTP-Met-tRNA ternary complex and other initiation factors like eIF3 to the 40S subunit. To test the effect of nsp1 on 48S complex formation, in which the 43S complex binds to the mRNA, and 80S monosome assembly, we examined, by sucrose gradient fractionation, the levels of accumulation of these complexes on a radiolabeled mRNA template in the presence of nsp1. To analyze the effect of nsp1 on 80S formation, we incubated 32 P-labeled rLuc RNA template with nsp1, GST, or nsp1-mt in RRL in the presence of cycloheximide (CHX). CHX treatment inhibits the elongation step, but does not affect 80S monosome assembly15. We used hippuristanol, which blocks eIF4A function and inhibits 48S complex formation, as a control16. CHX treatment induced 80S complex accumulation and hippuristanol inhibited it (Fig. 3a) . Nsp1, but not GST and nsp1-mt, suppressed 80S complex formation (Fig. 3b) . We tested the effect of nsp1 on 48S complex formation by incubating the samples with GMP-PNP and subsequently, subjecting them to sucrose gradient centrifugation; GMP-PNP does not affect 48S complex formation, whereas it blocks GTP hydrolysis by eIF2 in eIF2-GTP-Met-tRNA complexes and inhibits the release of eIF2 and subsequent joining of the 60S ribosomal subunit17. The 48S complex accumulated in the presence of GMP-PNP, but not in the presence of hippuristanol (Fig. 3c) . Nsp1, nsp1-mt and GST did not suppress the 48S complex formation (Fig. 3d) . These data showed that nsp1 did not inhibit 48S complex formation, but it suppressed 60S subunit joining.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 978,\n",
            "                    \"end\": 987,\n",
            "                    \"text\": \"(Fig. 3a)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1054,\n",
            "                    \"end\": 1063,\n",
            "                    \"text\": \"(Fig. 3b)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1533,\n",
            "                    \"end\": 1542,\n",
            "                    \"text\": \"(Fig. 3c)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1610,\n",
            "                    \"end\": 1619,\n",
            "                    \"text\": \"(Fig. 3d)\",\n",
            "                    \"ref_id\": null\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Nsp1 inhibits 80S formation\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Next, we performed a toeprinting analysis, which defines the positions of stalled ribosomes on the mRNA chain18. After initial incubation of RRL with nsp1, GST or nsp1-mt in the presence of CHX or a mixture of CHX and GMP-PNP, we added the rLuc RNA that had been pre-annealed with a 5'-end labeled primer to the samples. Subsequently, we subjected the samples to primer extension experiments without extracting the RNAs. In control groups, using GST and nsp1-mt, we detected a correctly positioned toeprint (TP-AUG), which was a premature primer extension termination signal that was induced by the presence of the stalled 40S subunit. The toeprint was approximately 15-17 nt downstream from the translation initiation codon in the presence of GMP-PNP and CHX, which allow 48S complex formation, but not 60S joining (Fig. 4a , lanes 5 to 7). In the presence of CHX, the control groups showed an additional band approximately 3 nt downstream from TP-AUG, which was most probably generated by the joining of the 60S subunits with the 40S complex positioned at the AUG (Fig. 4a , asterisk, lanes 2 to 4). Consistent with the observation that nsp1 suppressed the 80S complex formation (Fig. 3b ), nsp1 caused a clear reduction in the amount of the toeprint for the 80S complex (Fig. 4a, lane 3) . Compared to the observations in the control groups, the abundance of the TP-AUG was also reduced in the sample containing nsp1 (Fig. 4a, lanes 3 and 6) , which suggested that nsp1 induced inefficient startcodon recognition. Unexpectedly, we detected several additional bands, both upstream and downstream of the TP-AUG, in the sample incubated with nsp1, but not in controls; a longer exposure of the CHX-treated sample clearly showed the same signals (data not shown). These data suggested that nsp1 induced the modification of rLuc RNA at several discrete sites, which blocked cDNA elongation on these modified RNAs.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 816,\n",
            "                    \"end\": 824,\n",
            "                    \"text\": \"(Fig. 4a\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1066,\n",
            "                    \"end\": 1074,\n",
            "                    \"text\": \"(Fig. 4a\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1181,\n",
            "                    \"end\": 1189,\n",
            "                    \"text\": \"(Fig. 3b\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1273,\n",
            "                    \"end\": 1290,\n",
            "                    \"text\": \"(Fig. 4a, lane 3)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1420,\n",
            "                    \"end\": 1444,\n",
            "                    \"text\": \"(Fig. 4a, lanes 3 and 6)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Characterization of the 48S complex by toeprinting analysis\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We performed primer extension analysis to determine whether nsp1 indeed induced template RNA modification. After incubation of rLuc RNA with GST, nsp1 or nsp1-mt in RRL, as described above, we extracted the RNA and performed primer extension analysis using the same 5'-end labeled primer. In addition to the full-length primer extension product, only samples incubated with nsp1 showed several premature primer extension termination products, all of which corresponded to those detected in the toeprinting analysis performed in the presence of nsp1 (Fig. 4b) . Northern blot analysis of rLuc RNA, extracted after incubation with nsp1 in RRL under the same conditions, did not show a significant reduction in the levels of full length rLuc RNA transcripts compared to the levels in the control groups ( Supplementary Fig. 2 ). These data demonstrated that nsp1 induced the modification of capped rLuc RNA at several sites in the 5'-region without inducing extensive template RNA degradation.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 549,\n",
            "                    \"end\": 558,\n",
            "                    \"text\": \"(Fig. 4b)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 802,\n",
            "                    \"end\": 822,\n",
            "                    \"text\": \"Supplementary Fig. 2\",\n",
            "                    \"ref_id\": null\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Nsp1 modifies and renders the capped mRNA translationally inactive\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"To further characterize the nsp1-induced template RNA modification, we incubated caplabeled rLuc RNA and 3'-end-labeled rLuc RNA, independently, with nsp1 in RRL in the presence of CHX and GMP-PNP; GST and nsp1-mt served as controls. After incubation, we analyzed the extracted RNAs on a 5% sequencing gel. The amounts of the intact cap-labeled rLuc RNA, which had been incubated with nsp1, were clearly lower than that of the controls (Figs. 5a and 5b), demonstrating that the 5'-end region of the capped rLuc RNA was removed in the presence of nsp1. Incubation of the 3'-end-labeled rLuc RNA with nsp1 resulted in the appearance of an additional band that migrated slightly faster than the intact RNA (Fig. 5a , asterisk). The total radioactivity of the intact 3'-end labeled rLuc RNA and the fast-migrating RNA band in nsp1-treated samples was similar to that of the intact 3'-end labeled rLuc RNA in the controls (Fig. 5b) . To determine the approximate size of the fastmigrating band, we incubated 3'-end-labeled non-polyadenylated rLuc RNA with nsp1 in RRL; we used non-polyadenylated rLuc RNA because some of the polyadenylated rLuc RNA preparations had mRNA species with shorter poly(A) tails, which interfered with the positive identification of the truncated RNA products in the gel. We observed two major fast-migrating bands and their sizes were approximately 20 to 40 nt shorter than the fulllength RNA template ( Supplementary Fig. 3 , asterisks). These data demonstrated that nsp1 induced the removal of the 5'-end, but not the 3'-end, from the rLuc RNA.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 703,\n",
            "                    \"end\": 711,\n",
            "                    \"text\": \"(Fig. 5a\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 917,\n",
            "                    \"end\": 926,\n",
            "                    \"text\": \"(Fig. 5b)\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1427,\n",
            "                    \"end\": 1447,\n",
            "                    \"text\": \"Supplementary Fig. 3\",\n",
            "                    \"ref_id\": null\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Nsp1 modifies and renders the capped mRNA translationally inactive\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Next, we tested the translational competence of the rLuc RNA that had undergone the nsp1induced modification. After incubation of the rLuc RNA in RRL in the presence of nsp1, GST or nsp1-mt, we added the same amount of CAT RNA to each sample as a spike RNA, extracted the RNAs and performed a second in vitro translation reaction without adding nsp1, GST or nsp1-mt. We performed this analysis to examine the translational competence of RNAs that were extracted from the first in vitro translation reaction. As expected, nsp1 suppressed the translation of rLuc RNA in the initial translation ( Fig. 5c 1 st , 5d ). The rLuc RNA that had undergone the nsp1-mediated RNA modification was not efficiently translated into the rLuc protein ( Fig. 5c 2 nd , 5d ), which demonstrated that the modified rLuc RNA was translationally inactive.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 594,\n",
            "                    \"end\": 611,\n",
            "                    \"text\": \"Fig. 5c 1 st , 5d\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 737,\n",
            "                    \"end\": 754,\n",
            "                    \"text\": \"Fig. 5c 2 nd , 5d\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Nsp1 modifies and renders the capped mRNA translationally inactive\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"In addition to cap-dependent translation, nsp1 also suppressed translation mediated by IRESs derived from HCV and CrPV ( Fig. 1) . To address the question whether the nsp1induced template RNA modification is specific for cap-dependent RNA template, we tested whether nsp1 induced the modification of IRES in RNA templates carrying HCV, CrPV and encephalomyocarditis virus (EMCV)-derived IRESs. We incubated the RNA transcripts, Ren-HCV-FF, Ren-CrPV-FF and Ren-EMCV-FF, a bicistronic RNA carrying EMCV IRES between the upstream rLuc ORF and the downstream fLuc ORF (Fig. 6a ), in RRL in the presence of nsp1, GST or nsp1-mt. After incubation, we extracted the RNAs and subjected them to Northern blot analysis (Fig. 6 ). Nsp1 did not induce any RNA cleavage in Ren-HCV-FF and Ren-CrPV-FF ( Fig. 6b and c, respectively) . In contrast, nsp1 induced an RNA cleavage in Ren-EMCV-FF (Fig. 6d) . A cleaved RNA fragment, detected by a probe that binds to the rLuc ORF, migrated similarly with the control RNA1, carrying the rLuc ORF and EMCV IRES (Fig. 6d) , while the size of the other RNA fragment, detected by a probe that binds to fLuc ORF, was similar to the size of fLuc ORF in Ren-EMCV-FF. These data suggested that nsp1 induced the cleavage at or near the 3'-region of the EMCV IRES in Ren-EMCV-FF (Fig. 6d) . These results demonstrated that nsp1 induced a template-dependent RNA cleavage in IRES-driven RNAs and that the nsp1-induced RNA modification was not specific for cap-dependent RNA template.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 121,\n",
            "                    \"end\": 128,\n",
            "                    \"text\": \"Fig. 1)\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 564,\n",
            "                    \"end\": 572,\n",
            "                    \"text\": \"(Fig. 6a\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 709,\n",
            "                    \"end\": 716,\n",
            "                    \"text\": \"(Fig. 6\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 789,\n",
            "                    \"end\": 817,\n",
            "                    \"text\": \"Fig. 6b and c, respectively)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 877,\n",
            "                    \"end\": 886,\n",
            "                    \"text\": \"(Fig. 6d)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1039,\n",
            "                    \"end\": 1048,\n",
            "                    \"text\": \"(Fig. 6d)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1298,\n",
            "                    \"end\": 1307,\n",
            "                    \"text\": \"(Fig. 6d)\",\n",
            "                    \"ref_id\": null\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Nsp1 induces RNA cleavage in an IRES-driven mRNA template\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"To assess the biological significance of nsp1 binding to 40S subunits, we examined whether nsp1 could modify the rLuc RNA without ribosomes. After pelleting down the ribosomes in RRL by centrifugation, we incubated the ribosome-free supernatant with the cap-labeled rLuc RNA and nsp1 in the presence of CHX and GMP-PNP; GST and nsp1-mt served as controls. Gel electrophoresis analysis of the extracted RNAs showed that the amounts of the cap-labeled rLuc RNA were similar among the three samples ( Fig. 7a and 7b ). To know whether the rLuc RNA that had been incubated with nsp1 in the absence of the ribosomes was translationally competent, we incubated the capped rLuc RNA with nsp1, GST, or nsp1mt in the ribosome-free RRL. As expected, rLuc protein synthesis did not occur in the ribosome-free lysate (Fig. 7c, top panel) . After incubation, we added the same amount of spike CAT RNA to the samples, extracted the RNAs and subjected them to in vitro translation reaction without adding nsp1, GST or nsp1-mt. The rLuc RNA that had been incubated with nsp1 in the ribosome-free RRL was translationally competent in the second in vitro translation assay (Fig. 7c, middle panel) . These data suggested that the 40S subunitbound nsp1 exerted the modification and inactivation of rLuc RNA.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 498,\n",
            "                    \"end\": 512,\n",
            "                    \"text\": \"Fig. 7a and 7b\",\n",
            "                    \"ref_id\": \"FIGREF3\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 805,\n",
            "                    \"end\": 825,\n",
            "                    \"text\": \"(Fig. 7c, top panel)\",\n",
            "                    \"ref_id\": \"FIGREF3\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1155,\n",
            "                    \"end\": 1178,\n",
            "                    \"text\": \"(Fig. 7c, middle panel)\",\n",
            "                    \"ref_id\": \"FIGREF3\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"The nsp1-40S subunit complex exerts template RNA modification\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We further tested whether the association of rLuc RNA with the 43S complex was required for nsp1-mediated template RNA modification by incubating the cap-labeled rLuc RNA with nsp1, GST or nsp1-mt in RRL in the presence of hippuristanol. Nsp1 did not affect the amount of the labeled RNA in the presence of hippuristanol (Fig. 7d) , leading to our suggestion that the modification of the capped RNA required the association of template RNA with the nsp1-40S complex.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 321,\n",
            "                    \"end\": 330,\n",
            "                    \"text\": \"(Fig. 7d)\",\n",
            "                    \"ref_id\": \"FIGREF3\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"The nsp1-40S subunit complex exerts template RNA modification\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Nsp1 suppressed the translation mediated by CrPV IRES (Fig. 1 ) but did not induce the RNA cleavage in Ren-CrPV-FF (Fig. 6b) . These data suggested that in addition to template RNA modification, the association of nsp1 with 40S ribosomal subunit also inactivated the translational function of 40S ribosomes. To further examine this possibility, we subjected Ren-CrPV-FF to in vitro translation in RRL in the presence of nsp1, GST or nsp1-mt. After incubation, we analyzed the aliquots of the samples by SDS-PAGE. As expected, nsp1, but not GST or nsp1-mt, suppressed both cap-dependent and IRES-dependent translation (Fig.  8a, 1 st ) . We added the same amount of the spike CAT RNA to the remaining samples, extracted RNAs and performed a second in vitro translation analysis without adding nsp1, GST or nsp1-mt (Fig. 8a, 2 nd , 8b) . The cap-dependent rLuc gene translation, from Ren-CrPV-FF RNA that was exposed to nsp1 in the first in vitro translation reaction, was inhibited but the IRES-mediated fLuc translation was not inhibited. Thus, nsp1 suppressed CrPV IRES-mediated translation in the first in vitro translation reaction but did not induce the modification that inactivated the translational activity of CrPV IRES in the second in vitro translation assay. Like rLuc RNA (Fig. 5) , incubation of cap-labeled Ren-CrPV-FF RNA with nsp1 resulted in the removal of the 5'-end region from this RNA (Fig. 8c) . We interpreted these results to suggest that binding of nsp1 to 40S ribosomal subunits inactivated the translational functions of 40S ribosomes leading to the inhibition of CrPV IRES-driven translation in Ren-CrPV-FF. Collectively, these data showed that nsp1 suppressed translation by inactivating the translational functions of 40S ribosomal subunits and modifying the template RNA.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 54,\n",
            "                    \"end\": 61,\n",
            "                    \"text\": \"(Fig. 1\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 115,\n",
            "                    \"end\": 124,\n",
            "                    \"text\": \"(Fig. 6b)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 617,\n",
            "                    \"end\": 634,\n",
            "                    \"text\": \"(Fig.  8a, 1 st )\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 813,\n",
            "                    \"end\": 833,\n",
            "                    \"text\": \"(Fig. 8a, 2 nd , 8b)\",\n",
            "                    \"ref_id\": \"FIGREF4\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1284,\n",
            "                    \"end\": 1292,\n",
            "                    \"text\": \"(Fig. 5)\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1406,\n",
            "                    \"end\": 1415,\n",
            "                    \"text\": \"(Fig. 8c)\",\n",
            "                    \"ref_id\": \"FIGREF4\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"The nsp1-40S subunit complex is translationally inactive\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Host translational suppression that is often detected in virus-infected cells is an effective viral defensive strategy for suppressing host innate immune responses and freeing host translational machinery for viral-specific gene expression. Although many viral proteins that suppress host translation have been identified19-22, SCoV nsp1 8 and several herpes virus proteins23-26, including herpes simplex virus-1 vhs protein27, are known to promote destabilization of host mRNAs and suppress host translation. Vhs protein, which has a sequence similarity with known RNases, is indeed an RNase,28 induces RNA cleavage in RRL29, and degrades both host and viral mRNAs30. Unlike vhs, none of the known proteins share similarities with SCoV nsp1 in their sequence and structure. The biological activities of SCoV nsp1 also substantially differed from those of vhs, as nsp1 inactivated the translational functions of the 40S ribosomal subunit and modified RNA transcripts only in the context of its association with the 40S subunit. The strategy that SCoV nsp1 has evolved to bind to 40S subunits to exert its function appears to be an excellent one for blocking translation of both capped mRNAs and IRES-containing mRNAs, and for efficiently accessing host mRNAs, including those involved in host innate immune functions, and inactivating them.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Discussion\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"By binding to the 40S ribosomal subunits (Fig. 2, Supplementary Fig. 1 ) and inactivating them (Fig. 8) , SCoV nsp1 efficiently suppressed cap-dependent and IRES-mediated translation, including translation mediated by CrPV IRES. In contrast to SCoV nsp1, CrPV IRES-mediated translation is relatively insensitive to Drosophila Reaper, a a potent apoptotic inducer, which also inhibits translation by binding directly to the 40S subunit31.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 41,\n",
            "                    \"end\": 70,\n",
            "                    \"text\": \"(Fig. 2, Supplementary Fig. 1\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 95,\n",
            "                    \"end\": 103,\n",
            "                    \"text\": \"(Fig. 8)\",\n",
            "                    \"ref_id\": \"FIGREF4\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Discussion\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Further studies are needed to determine how the binding of nsp1 to 40S ribosomal subunit inactivates their translational function. Using rLuc RNA as a template, we showed that the formation of the 80S complex, but not the 48S complex was inhibited in the presence of nsp1 (Fig. 3) . Because the nsp1-40S complex also modified the 5'-region of the rLuc RNA (Figs. 4 and 5) , it is possible that the nsp1-mediated RNA modification affected the formation of 48S and/or 80S complexes on these modified RNA templates (Fig. 3) . We obtained somewhat conflicting results, which showed that nsp1 did not suppress the 48S assembly on rLuc RNA (Fig. 3d) but the 48S TP-AUG signal was reduced in the presence of nsp1 (Fig. 4a) . These data may imply that some of the multiple toe-prints upstream of the TP-AUG, observed in nsp1-treated sample (Fig. 4a, lane 6) , represent arrested 48S translation intermediates as a result of nsp1 inactivating the function of 40S ribosomal subunit leading to inefficient start site recognition; these arrested 48S translation intermediates could contribute to the signal observed for 48S complex formation (Fig. 3d) . Furthermore, the nsp1-induced RNA modification alone cannot fully account for the substantially reduced luciferase protein synthesis from capped rLuc RNA (Fig. 1) , highlighting the contribution of the translation inhibition function of nsp1 towards the suppression of reporter protein expression in RRL. To clarify the exact step in the translation of capped mRNAs that nsp1 inhibits, it would be useful to isolate an nsp1 mutant that retains the 40S subunit inactivating function but lacks the RNA modification function, and test the effect of this mutant on 48S and 80S complex formations.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 272,\n",
            "                    \"end\": 280,\n",
            "                    \"text\": \"(Fig. 3)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 356,\n",
            "                    \"end\": 371,\n",
            "                    \"text\": \"(Figs. 4 and 5)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 512,\n",
            "                    \"end\": 520,\n",
            "                    \"text\": \"(Fig. 3)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 634,\n",
            "                    \"end\": 643,\n",
            "                    \"text\": \"(Fig. 3d)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 706,\n",
            "                    \"end\": 715,\n",
            "                    \"text\": \"(Fig. 4a)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 832,\n",
            "                    \"end\": 849,\n",
            "                    \"text\": \"(Fig. 4a, lane 6)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1130,\n",
            "                    \"end\": 1139,\n",
            "                    \"text\": \"(Fig. 3d)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1296,\n",
            "                    \"end\": 1304,\n",
            "                    \"text\": \"(Fig. 1)\",\n",
            "                    \"ref_id\": \"FIGREF0\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Discussion\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"In addition to the modification of the 5'-region of capped rLuc RNA (Fig. 5) , Nsp1 also inactivated the translational competence of capped RNAs encoding CAT gene or fLuc gene (data not shown), suggesting that nsp1 could modify capped RNAs in general. Nsp1 induced a substantial reduction in the amount of full-length cap-labeled rLuc RNA (Fig. 5a) , which suggested a possibility that the very 5'-end of the rLuc RNA, including the 7methylguanosine cap and the radiolabeled phosphate group, was removed. Primer extension analysis of the rLuc RNA that had been incubated with nsp1 (Fig. 4b) revealed that nsp1 exerted the modification in several different regions at the 5'-end of rLuc RNA. An RNA signal that migrated slightly faster than the full-length rLuc RNA was detected after incubation of the 3'-end labeled rLuc RNA in the presence of nsp1 (Fig. 5a) . Analysis of the nsp1-treated non-polyadenylated rLuc RNA further showed that the fast migrating species of rLuc RNA shown in Fig. 5a consisted of at least two bands that were roughly 20 to 40 nt shorter than the full-length rLuc RNA template ( Supplementary Fig. 3 ). These data point to a possibility that the several premature primer extension termination products, which were obtained using nsp1-treated rLuc RNA (Fig. 4b) , were derived from rLuc RNA species lacking different regions from their 5'-ends. Because nsp1 induced an internal RNA cleavage in Ren-EMCV-FF RNA (Fig. 6) , we suspect that nsp1 also triggered a similar endonucleolytic RNA cleavage in the 5' region of rLuc RNA. We also noted that the amounts of the full-length primer extension product, obtained from rLuc RNA that had been incubated with nsp1, were not substantially different from the corresponding signal obtained from the control group (Fig. 4b) . These data imply that nsp1 induced the removal of the entire or a portion of the cap structure without affecting the first 5'-end nucleotide from the full-length rLuc RNA. Probably, the putative nsp1-induced endonucleolytic cleavage and decapping activities both contributed to the translational inactivation of nsp1-treated rLuc RNA.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 68,\n",
            "                    \"end\": 76,\n",
            "                    \"text\": \"(Fig. 5)\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 339,\n",
            "                    \"end\": 348,\n",
            "                    \"text\": \"(Fig. 5a)\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 581,\n",
            "                    \"end\": 590,\n",
            "                    \"text\": \"(Fig. 4b)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 850,\n",
            "                    \"end\": 859,\n",
            "                    \"text\": \"(Fig. 5a)\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 987,\n",
            "                    \"end\": 994,\n",
            "                    \"text\": \"Fig. 5a\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1106,\n",
            "                    \"end\": 1126,\n",
            "                    \"text\": \"Supplementary Fig. 3\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1278,\n",
            "                    \"end\": 1287,\n",
            "                    \"text\": \"(Fig. 4b)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1436,\n",
            "                    \"end\": 1444,\n",
            "                    \"text\": \"(Fig. 6)\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 1781,\n",
            "                    \"end\": 1790,\n",
            "                    \"text\": \"(Fig. 4b)\",\n",
            "                    \"ref_id\": \"FIGREF1\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Discussion\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Nsp1 induced RNA cleavage in the EMCV IRES of Ren-EMCV-FF RNA but it did not induce the RNA cleavage in Ren-CrPV-FF and Ren-HCV-FF RNAs, revealing the templatedependent nature of nsp1-induced RNA cleavage. Furthermore, these data demonstrated that nsp1-induced RNA modification was not limited to cap-dependent RNA templates. Because the translation initiation factors, eIF4G and eIF4A, are required for cap-dependent and EMCV IRES-mediated translation but not for translation driven by the IRESs of HCV and CrPV32, these factors may be involved in the induction of nsp1-mediated endonucleolytic RNA cleavage of template RNAs. Alternatively, nsp1 may recognize a putative unique RNA element(s), which is present in EMCV IRES and is absent in HCV and CrPV IRESs, to induce the RNA cleavage in the EMCV IRES. Further studies using various RNAs carrying different types of IRESs will be necessary to distinguish between these two possibilities.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Discussion\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Two different mechanisms are conceivable as to how the nsp1-40S ribosome complex induces the RNA modification at the 5'-end region of capped rLuc RNA and at the EMCV IRES of Ren-EMCV-FF RNA. One is that the activated nsp1, induced by the binding of nsp1 to the 40S subunits, exerts the modification of these RNAs. Another possibility is that binding of nsp1 to the 40S subunits induces the activation or recruitment of a host factor(s) that modifies the template RNAs. It is also possible that some ribosome-associated factors are involved in nsp1-induced RNA modification. In fact, host endonucleases can destabilize host mRNAs during translational elongation in polysomes33,34. Because nsp1 modified the rLuc RNA during translation initiation (Figs. 3-5) , the nsp1-mediated modification of the 5'region of the capped RNA presents a novel strategy for modifying capped RNAs.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 745,\n",
            "                    \"end\": 756,\n",
            "                    \"text\": \"(Figs. 3-5)\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Discussion\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"The expression of nsp1 in mammalian cells caused both host mRNA degradation and host translation inhibition8. Because host mRNAs that are not involved in active translation are delivered to host mRNA degradation pathways35, we hypothesize that host capped mRNAs that underwent the nsp1-mediated translation inhibition and modification in expressing cells and SCoV-infected cells are delivered to the host mRNA degradation machinery36 and efficiently degraded.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Discussion\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Efficient SCoV-specific gene expression occurs in infected cells, in which nsp1 suppressed host gene expression10, which implied that SCoV gene expression is somehow immune to the nsp1-induced suppression of gene expression in infected cells. Like many host mRNAs, coronavirus mRNAs are capped and polyadenylated37. Several mechanisms are conceivable for an apparent selective immunity of SCoV gene expression to the nsp1-induced translational suppression. One is that the SCoV-specific translation occurs in selected subcellular compartments, where the localization and the access of nsp1 is limited. Another possibility is that a viral or host-derived protein(s) selectively binds to SCoV mRNAs and facilitates SCoV-specific translation in the presence of nsp1. Alternatively, the efficient accumulation of SCoV mRNAs outcompetes the nsp1-induced translational suppression.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Discussion\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We cloned pST-fLuc by inserting the cDNA encoding the entire fLuc ORF region from the pGL3 control vector (Promega) into pSPT18 (Roche). We cloned the PCR-amplified fulllength nsp1 and nsp1-mt genes into the pGEX vector (GE Healthcare), yielding pGEX-wt and pGEX-mt, respectively. We constructed pRL-EMCV-FL by replacing the fragment containing HCV IRES, fLuc gene and the 3'-noncoding region of pRC22F38 with that containing EMCV IRES, fLuc gene and the 3'-noncoding region of pT7-IRES-fLuc39.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Plasmid construction\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We generated capped and polyadenylated CAT RNA, nsp1 RNA, nsp1-mt RNA, m 7 G-FF RNA, rLuc RNA, Ren-HCV-FF RNA, and Ren-CrPV-FF RNA by linearizing and transcribing the plasmids pcD-CAT, pcD-Nsp1-wt, pcD-Nsp1-mt10, pST-fluc, pRL-SV40 (Promega), pRC22F38, and pGL3/Ren/CrPV/FF40, respectively, using an mMESSAGE mMACHINE T7 Ultra kit (Ambion). We synthesized capped and non-polyadenylated Ren-EMCV-FF RNA from pRL-EMCV-FL after linearization by Cla I digestion at 288 nt upstream of the 3'-end of fLuc gene. We used TransIT mRNA (Mirus) for transfecting in vitro RNA transcripts into subconfluent 293 cells.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"In vitro RNA synthesis and RNA transfection\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We purified GST-fused proteins expressed in E. coli BL21-CodonPlus DE3 cells (Stratagene) using glutathione Sepharose 4B affinity beads (GE Healthcare) followed by cleavage of GST from the GST-fused protein using PreScission protease (GE Healthcare).\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Expression and purification of recombinant nsp1 protein\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We performed the translation reactions using Retic Lysate IVT\\u2122 Kit (Ambion). After preincubating RRL with nsp1, GST or nsp1-mt for 10 min at 4\\u00b0C, we added RNA and incubated the samples at 30\\u00b0C in the presence of [ 35 S]methionine (1,000 Ci/mmol; MP Biomedicals). We resolved the translation products by 12% SDS-PAGE and visualized by autoradiography. We performed densitometric analysis using ImageJ software to quantify the bands. .\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"In vitro translation assay\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"At 8 h post transfection of 293 cells with RNAs, we prepared the cell lysates using a buffer containing 50 mM Tris-HCl (pH 7.5), 5 mM MgCl 2 , 100 mM KCl, and 1% (v/v) Triton-X-100 and incubated the lysates with an anti-myc antibody (Upstate Biotech.) at 4\\u00b0C. We collected the immune complexes after incubation with Protein G Plus-Agarose (Santa Cruz Biotech.) at 4\\u00b0C followed by centrifugation. After washing the pellets three times in a buffer containing 10 mM HEPES (pH 7.4), 500 mM KCl, 2.5 mM MgCl 2 and 1 mM DTT, we extracted RNA and performed Northern blot analysis using digoxigenin-labeled oligonucleotide probes41 to detect 18S and 28S rRNAs. We resolved the immunoprecipitated proteins in the pellets by SDS-PAGE and Western blot analysis.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Immunoprecipitation of rRNA\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"At 8 h post transfection of 293 cells with RNAs, we prepared the cell lysates in a buffer containing 50 mM Tris-HCl (pH 7.5), 5 mM MgCl 2 , 100 mM KCl, 1% (v/v) Triton-X-100, 2 mM DTT, 100 \\u03bcg ml \\u22121 CHX, and 0.5 mg ml \\u22121 heparin, applied them onto a 10% to 50% sucrose gradient containing the same buffer and centrifuged at 39,000 rpm in a Beckman SW41 rotor at 4\\u00b0C for 2 h. After fractionating the gradient fractions using a gradient fractionator, we monitored the absorbance at 254 nm using an ISCO UA-6 spectrophotometer. We precipitated the proteins from each fraction using trichloroacetic acid and detected them by Western blot analysis. We extracted rRNAs from the fractions and visualized them on a gel by staining with ethidium bromide.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Polysome analysis\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We performed the ribosome binding assay as described previously16. After preincubating RRL with GST, nsp-1, or nsp1-mt for 10 min at 4\\u00b0C, we incubated the samples with [\\u03b1-32 P] UTP-labeled, in vitro transcribed rLuc RNA in the presence of 0.6 mM CHX or 1 mM GMP-PNP for 10 min at 30\\u00b0C. After centrifugation through 10 to 40% sucrose gradients (38,000 rpm for 3.5 h in a Beckman SW41 rotor), we collected fractions from each gradient and determined the radioactivity in each fraction in a scintillation counter.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Ribosome binding assay\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We performed toe printing analysis as described previously18,42. After pre-annealing the 5' 32 P-labeled primer DNA (5'-TTT TTC TGA ATC ATA ATA ATT AA-3') (40,000 c.p.m.) to rLuc RNA by heating for 1 min at 65\\u00b0C, followed by incubation at 37\\u00b0C for 10 min, we preincubated RRL with 1 \\u03bcg of GST, nsp1, or nsp1-mt for 10 min at 4\\u00b0C and then incubated the samples with 0.6 mM CHX or 0.6 mM CHX and 1 mM GMP-PNP for 5 min at 30\\u00b0C. After incubation, we added the primer-mRNA complexes to the samples and incubated for an additional 10 min at 30\\u00b0C. We diluted the reactions 20-fold with buffer containing 0.5 mM CHX, 7 mM magnesium acetate, 100 mM potassium acetate, 0.5 mM dATP, dCTP, dGTP, and dTTP, 20 mM Tris-HCl (pH 7.4), 2 mM DTT, and 1 unit/\\u03bcl SuperScript III (invitrogen) and incubated them at 30\\u00b0C for 10 min. We resolved the extracted primer extension products on an 8% sequencing gel.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Toe printing analysis\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"After preincubating RRL with GST, nsp-1, or nsp-1-mt for 10min at 4\\u00b0C, we incubated the samples with 0.6 mM CHX or 0.6 mM CHX and 1 mM GMP-PNP for 5 min at 30\\u00b0C. Then, we added rLuc RNA to the samples and incubated them for 10 min at 30\\u00b0C. After extracting total RNAs, we incubated the RNAs with the 5'-end labeled primer as described above and performed primer extension using a Primer Extension System (Promega). We resolved the extracted primer extension products on 8% sequencing gel.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Primer extension analysis\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"We generated the cap-labeled rLuc RNA by incubating uncapped rLuc RNA with vaccinia virus guanylyltransferase (Ambion) in the presence of [\\u03b1-32 P]GTP (3,000 Ci/mmol; MP Biomedicals) at 37\\u00b0C for 1 h. We generated the 3'-end labeled rLuc RNA by incubating rLuc RNA with T4 RNA ligase (Ambion) in the presence of [ 32 P]pCp (cytidine-3', 5' bisphosphate; 3000 Ci/mmol, MP biomedicals) and ATP at 4\\u00b0C for 16 h. We purified all labeled RNA transcripts with the RNeasy kit (Qiagen Inc.). We incubated the [ 32 P]-labeled RNA transcripts with nsp1, GST or nsp1-mt in RRL. After 15 min incubation at 30\\u00b0C in the presence of CHX and GMP-PNP, we extracted the RNAs and resolved them on 5% sequencing gel.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Assay for modification of radiolabeled in vitro-synthesized RNAs in RRL\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Refer to Web version on PubMed Central for supplementary material. Translated products were analyzed on SDS-PAGE and detected by autoradiography. (e) For a given RNA group, the amount of the firefly luc protein, which was determined by densitometric analysis, in each experimental group was normalized to the amount of the fLuc protein in a control group, to which the purified protein was not added. Each data point represents the average of three independent experiments and the standard deviation is shown.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Supplementary Material\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"(a-c) 293 cells were independently transfected with CAT RNA (a), nsp1 RNA (b), or nsp1mt RNA (c). Cell extracts were prepared at 8 h after transfection and subjected to polysome profile analysis (top panels). CAT, nsp1 (wt) and nsp1-mt (mt) in each fraction were detected by Western blot analysis using anti-myc antibody (middle panels). Bottom panels show rRNAs in each fraction. (d-f) After incubation of the mixture of 0.25 \\u03bcg of rLuc RNA and 1 \\u03bcg of nsp1, GST or nsp1-mt in RRL for 30 min at 30\\u00b0C, the samples were separated on 10-50% sucrose gradient. GST, nsp1 (wt) and nsp1-mt (mt) in each fraction were detected by Western blot analysis using anti-GST antibody and anti-nsp1 8 antibody. The locations of GST (d), nsp1 (e) or nsp1-mt (f) in fractions are shown (all top panels). The bottom panels show rRNAs. (g-i) 293 cells were transfected with CAT RNA (CAT), nsp1 RNA (wt), or nsp1-mt RNA (mt). Cell extracts were prepared at 8 h after transfection and subjected to immunoprecipitation using anti-myc antibody. Extracted RNAs from the immunoprecipitates were separated by agarose gel electrophoresis, and rRNAs were detected by Northern blot analysis (g). The immunoprecipitated proteins were examined by Western blot analysis using anti-S6 antibody (40S subunit specific) and anti-His antibody for CAT, nsp1 and nsp1-mt (h). Panel I represents the values, each of which was obtained by dividing the abundance of the immunoprecipitated 18S rRNA with the immunoprecipitated CAT protein, nsp1 protein or nsp1-mt protein in an arbitrary scale.\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Fig. 2. Binding of nsp1 to the 40S ribosomal subunits\"\n",
            "        },\n",
            "        {\n",
            "            \"text\": \"Ribosome binding assays of radiolabeled rLuc RNA was performed as described in the text. Each panel shows a representative data from three independent experiments. (a) rLuc RNA was incubated in RRL in the presence of CHX only or a mixture of CHX and hippuristanol (hipp). The percentages of the rLuc RNA bound in 80S complexes were: CHX, 17.4%; CHX + hipp., 1.0%. (b) rLuc RNA was incubated in RRL with GST, nsp1 (wt) or nsp1-mt (mt) in the presence of CHX. The percentages of the rLuc RNA bound in 80S complexes were: GST, 13.3%; nsp1, 5.4%; nsp1-mt, 14.8%. (c) rLuc RNA was incubated in RRL in the presence of GMP-PNP or a mixture of GMP-PNP and hipp. The percentages of the rLuc RNA bound in 48S complexes were: GMP-PNP, 11.7%; GMP-PNP + hipp., 2.9%. (d) rLuc RNA was incubated in RRL with GST, nsp1 (wt) or nsp1-mt (mt), in the presence of GMP-PNP. The percentages of the rLuc RNA bound in 48S complexes were: GST, 13.1%; nsp1, 12.1%; nsp1-mt, 12.0%. Kamitani (a) Samples shown in lanes 2-4 and 5-7 were incubated with CHX and a mixture of CHX and GMP-PNP, respectively, and toe printing analysis was performed in the presence of GST (lanes 2 and 5), nsp1 (lanes 3 and 6) or nsp1-mt (lanes 4 and 7). Lane 1, the primer extension products using the rLuc RNA that was hybridized with a 5'-end labeled primer; 5' end, the primer extension product that was extended to the 5'-end of rLuc RNA; UTR, untranslated region; TP-AUG, a correctly positioned toeprint. An arrowhead and asterisk represent a signal that was generated by the joining of the 60S subunits to the 40S complex positioned on the AUG. A dideoxynucleotide sequence of the rLuc gene generated with the same primer was run in parallel (left four rows). Translation initiation codon is shown at the left side of the gel. (b) RRL was incubated with GST (lanes 1 and 4), nsp1 (lanes 2 and 5), or nsp1-mt (lanes 3 and 6) in the presence of CHX (lanes 1 to 3) or a mixture of CHX and GMP-PNP (lanes 4 to 6) for 5 min at 30\\u00b0C. Then, rLuc RNA was added to each sample, and the samples were incubated for an additional 10 min at 30\\u00b0C. The RNAs were extracted and subjected to primer extension analysis. Lane 7, the primer extension products using the rLuc RNA that was hybridized with a 5'-end labeled primer. Kamitani Experiments were performed as described in Fig. 5c , except that Ren-CrPV-FF RNA was used as the template RNA. rLuc and fLuc are expressed by cap-dependent and CrPV IRESmediated mechanisms, respectively (a, 2 nd ). (b) Experiments described in (a) were repeated three times and the abundance of fLuc and rLuc proteins in the second (2 nd ) translation reaction is shown. (c) Cap-labeled Ren-CrPV-FF RNA was incubated with GST, nsp1, or nsp1-mt in RRL for 15 min at 30\\u00b0C in the presence of CHX and GMP-PNP. RNAs were extracted, resolved on a sequencing gel and visualized by autoradiography.\",\n",
            "            \"cite_spans\": [\n",
            "                {\n",
            "                    \"start\": 955,\n",
            "                    \"end\": 963,\n",
            "                    \"text\": \"Kamitani\",\n",
            "                    \"ref_id\": null\n",
            "                },\n",
            "                {\n",
            "                    \"start\": 2265,\n",
            "                    \"end\": 2273,\n",
            "                    \"text\": \"Kamitani\",\n",
            "                    \"ref_id\": null\n",
            "                }\n",
            "            ],\n",
            "            \"ref_spans\": [\n",
            "                {\n",
            "                    \"start\": 2317,\n",
            "                    \"end\": 2324,\n",
            "                    \"text\": \"Fig. 5c\",\n",
            "                    \"ref_id\": \"FIGREF2\"\n",
            "                }\n",
            "            ],\n",
            "            \"section\": \"Fig. 3. Ribosome binding assays\"\n",
            "        }\n",
            "    ],\n",
            "    \"bib_entries\": {\n",
            "        \"BIBREF0\": {\n",
            "            \"ref_id\": \"b0\",\n",
            "            \"title\": \"The aetiology, origins, and diagnosis of severe acute respiratory syndrome\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"L\",\n",
            "                    \"middle\": [\n",
            "                        \"L\"\n",
            "                    ],\n",
            "                    \"last\": \"Poon\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"Y\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Guan\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [\n",
            "                        \"M\"\n",
            "                    ],\n",
            "                    \"last\": \"Nicholls\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"K\",\n",
            "                    \"middle\": [\n",
            "                        \"Y\"\n",
            "                    ],\n",
            "                    \"last\": \"Yuen\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [\n",
            "                        \"S\"\n",
            "                    ],\n",
            "                    \"last\": \"Peiris\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2004,\n",
            "            \"venue\": \"Lancet Infect Dis\",\n",
            "            \"volume\": \"4\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"663--71\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF1\": {\n",
            "            \"ref_id\": \"b1\",\n",
            "            \"title\": \"Mechanisms and enzymes involved in SARS coronavirus genome expression\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"V\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Thiel\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2003,\n",
            "            \"venue\": \"J Gen Virol\",\n",
            "            \"volume\": \"84\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"2305--2320\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF2\": {\n",
            "            \"ref_id\": \"b2\",\n",
            "            \"title\": \"Identification and characterization of severe acute respiratory syndrome coronavirus replicase proteins\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"E\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Prentice\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Mcauliffe\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"X\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Lu\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"K\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Subbarao\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"R\"\n",
            "                    ],\n",
            "                    \"last\": \"Denison\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2004,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"78\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"9977--86\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF3\": {\n",
            "            \"ref_id\": \"b3\",\n",
            "            \"title\": \"The papain-like protease of severe acute respiratory syndrome coronavirus has deubiquitinating activity\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"N\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Barretto\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2005,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"79\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"15189--98\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF4\": {\n",
            "            \"ref_id\": \"b4\",\n",
            "            \"title\": \"The nsp2 replicase proteins of murine hepatitis virus and severe acute respiratory syndrome coronavirus are dispensable for viral replication\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [\n",
            "                        \"L\"\n",
            "                    ],\n",
            "                    \"last\": \"Graham\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"A\",\n",
            "                    \"middle\": [\n",
            "                        \"C\"\n",
            "                    ],\n",
            "                    \"last\": \"Sims\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"S\",\n",
            "                    \"middle\": [\n",
            "                        \"M\"\n",
            "                    ],\n",
            "                    \"last\": \"Brockway\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [\n",
            "                        \"S\"\n",
            "                    ],\n",
            "                    \"last\": \"Baric\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"R\"\n",
            "                    ],\n",
            "                    \"last\": \"Denison\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2005,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"79\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"13399--411\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF5\": {\n",
            "            \"ref_id\": \"b5\",\n",
            "            \"title\": \"The papain-like protease from the severe acute respiratory syndrome coronavirus is a deubiquitinating enzyme\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"H\",\n",
            "                    \"middle\": [\n",
            "                        \"A\"\n",
            "                    ],\n",
            "                    \"last\": \"Lindner\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2005,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"79\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"15199--208\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF6\": {\n",
            "            \"ref_id\": \"b6\",\n",
            "            \"title\": \"Deubiquitination, a new function of the severe acute respiratory syndrome coronavirus papain-like protease?\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"T\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Sulea\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"H\",\n",
            "                    \"middle\": [\n",
            "                        \"A\"\n",
            "                    ],\n",
            "                    \"last\": \"Lindner\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"E\",\n",
            "                    \"middle\": [\n",
            "                        \"O\"\n",
            "                    ],\n",
            "                    \"last\": \"Purisima\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Menard\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2005,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"79\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"4550--4551\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF7\": {\n",
            "            \"ref_id\": \"b7\",\n",
            "            \"title\": \"Severe acute respiratory syndrome coronavirus nsp1 protein suppresses host gene expression by promoting host mRNA degradation\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"W\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Kamitani\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2006,\n",
            "            \"venue\": \"Proc Natl Acad Sci\",\n",
            "            \"volume\": \"103\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"12885--90\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF8\": {\n",
            "            \"ref_id\": \"b8\",\n",
            "            \"title\": \"Severe acute respiratory syndrome coronavirus evades antiviral signaling: role of nsp1 and rational design of an attenuated strain\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"G\"\n",
            "                    ],\n",
            "                    \"last\": \"Wathelet\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Orr\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"B\"\n",
            "                    ],\n",
            "                    \"last\": \"Frieman\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [\n",
            "                        \"S\"\n",
            "                    ],\n",
            "                    \"last\": \"Baric\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2007,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"81\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"11620--11653\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF9\": {\n",
            "            \"ref_id\": \"b9\",\n",
            "            \"title\": \"Severe acute respiratory syndrome coronavirus nsp1 suppresses host gene expression, including that of type I interferon, in infected cells\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"K\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Narayanan\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2008,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"82\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"4471--4480\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF10\": {\n",
            "            \"ref_id\": \"b10\",\n",
            "            \"title\": \"Coronavirus non-structural protein 1 is a major pathogenicity factor: implications for the rational design of coronavirus vaccines\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Zust\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2007,\n",
            "            \"venue\": \"PLoS Pathog\",\n",
            "            \"volume\": \"3\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF11\": {\n",
            "            \"ref_id\": \"b11\",\n",
            "            \"title\": \"Factorless ribosome assembly on the internal ribosome entry site of cricket paralysis virus\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"E\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Jan\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"P\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Sarnow\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2002,\n",
            "            \"venue\": \"J Mol Biol\",\n",
            "            \"volume\": \"324\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"889--902\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF12\": {\n",
            "            \"ref_id\": \"b12\",\n",
            "            \"title\": \"Position of the CrPV IRES on the 40S subunit and factor dependence of IRES/80S ribosome assembly\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"T\",\n",
            "                    \"middle\": [\n",
            "                        \"V\"\n",
            "                    ],\n",
            "                    \"last\": \"Pestova\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"I\",\n",
            "                    \"middle\": [\n",
            "                        \"B\"\n",
            "                    ],\n",
            "                    \"last\": \"Lomakin\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"C\",\n",
            "                    \"middle\": [\n",
            "                        \"U\"\n",
            "                    ],\n",
            "                    \"last\": \"Hellen\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2004,\n",
            "            \"venue\": \"EMBO Rep\",\n",
            "            \"volume\": \"5\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"906--919\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF13\": {\n",
            "            \"ref_id\": \"b13\",\n",
            "            \"title\": \"Assays for eukaryotic protein synthesis\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"W\",\n",
            "                    \"middle\": [\n",
            "                        \"C\"\n",
            "                    ],\n",
            "                    \"last\": \"Merrick\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 1979,\n",
            "            \"venue\": \"Methods Enzymol\",\n",
            "            \"volume\": \"60\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"108--131\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF14\": {\n",
            "            \"ref_id\": \"b14\",\n",
            "            \"title\": \"Inhibitors of protein synthesis identified by a high throughput multiplexed translation screen\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"O\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Novac\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"A\",\n",
            "                    \"middle\": [\n",
            "                        \"S\"\n",
            "                    ],\n",
            "                    \"last\": \"Guenier\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Pelletier\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2004,\n",
            "            \"venue\": \"Nucleic Acids Res\",\n",
            "            \"volume\": \"32\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"902--917\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF15\": {\n",
            "            \"ref_id\": \"b15\",\n",
            "            \"title\": \"Functional characterization of IRESes by an inhibitor of the RNA helicase eIF4A\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"E\"\n",
            "                    ],\n",
            "                    \"last\": \"Bordeleau\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2006,\n",
            "            \"venue\": \"Nat Chem Biol\",\n",
            "            \"volume\": \"2\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"213--233\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF16\": {\n",
            "            \"ref_id\": \"b16\",\n",
            "            \"title\": \"Molecular mechanisms of translation initiation in eukaryotes\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"T\",\n",
            "                    \"middle\": [\n",
            "                        \"V\"\n",
            "                    ],\n",
            "                    \"last\": \"Pestova\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2001,\n",
            "            \"venue\": \"Proc Natl Acad Sci U S A\",\n",
            "            \"volume\": \"98\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"7029--7065\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF17\": {\n",
            "            \"ref_id\": \"b17\",\n",
            "            \"title\": \"Primer extension analysis of eukaryotic ribosome-mRNA complexes\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Kozak\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 1998,\n",
            "            \"venue\": \"Nucleic Acids Res\",\n",
            "            \"volume\": \"26\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"4853--4862\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF18\": {\n",
            "            \"ref_id\": \"b18\",\n",
            "            \"title\": \"Rabies virus matrix protein interplay with eIF3, new insights into rabies virus pathogenesis\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"A\",\n",
            "                    \"middle\": [\n",
            "                        \"V\"\n",
            "                    ],\n",
            "                    \"last\": \"Komarova\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2007,\n",
            "            \"venue\": \"Nucleic Acids Res\",\n",
            "            \"volume\": \"35\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"1522--1532\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF19\": {\n",
            "            \"ref_id\": \"b19\",\n",
            "            \"title\": \"Cytopathogenesis and inhibition of host gene expression by RNA viruses\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"D\",\n",
            "                    \"middle\": [\n",
            "                        \"S\"\n",
            "                    ],\n",
            "                    \"last\": \"Lyles\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2000,\n",
            "            \"venue\": \"Microbiol Mol Biol Rev\",\n",
            "            \"volume\": \"64\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"709--733\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF20\": {\n",
            "            \"ref_id\": \"b20\",\n",
            "            \"title\": \"Measles virus N protein inhibits host translation by binding to eIF3-p40\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"H\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Sato\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2007,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"81\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"11569--76\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF21\": {\n",
            "            \"ref_id\": \"b21\",\n",
            "            \"title\": \"Regulation of host cell translation by viruses and effects on cell function\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"S\",\n",
            "                    \"middle\": [\n",
            "                        \"R\"\n",
            "                    ],\n",
            "                    \"last\": \"Thompson\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"P\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Sarnow\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2000,\n",
            "            \"venue\": \"Curr Opin Microbiol\",\n",
            "            \"volume\": \"3\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"366--70\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF22\": {\n",
            "            \"ref_id\": \"b22\",\n",
            "            \"title\": \"Site-directed mutagenesis of the virion host shutoff gene (UL41) of herpes simplex virus (HSV): analysis of functional differences between HSV type 1 (HSV-1) and HSV-2 alleles\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"D\",\n",
            "                    \"middle\": [\n",
            "                        \"N\"\n",
            "                    ],\n",
            "                    \"last\": \"Everly\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"G\",\n",
            "                    \"middle\": [\n",
            "                        \"S\"\n",
            "                    ],\n",
            "                    \"last\": \"Read\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 1999,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"73\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"9117--9146\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF23\": {\n",
            "            \"ref_id\": \"b23\",\n",
            "            \"title\": \"Lytic KSHV infection inhibits host gene expression by accelerating global mRNA turnover\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"B\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Glaunsinger\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"D\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Ganem\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2004,\n",
            "            \"venue\": \"Mol Cell\",\n",
            "            \"volume\": \"13\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"713--736\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF24\": {\n",
            "            \"ref_id\": \"b24\",\n",
            "            \"title\": \"Host shutoff during productive Epstein-Barr virus infection is mediated by BGLF5 and may contribute to immune evasion\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Rowe\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2007,\n",
            "            \"venue\": \"Proc Natl Acad Sci U S A\",\n",
            "            \"volume\": \"104\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"3366--71\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF25\": {\n",
            "            \"ref_id\": \"b25\",\n",
            "            \"title\": \"Pathogenesis of herpes simplex virus type 2 virion host shutoff (vhs) mutants\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"T\",\n",
            "                    \"middle\": [\n",
            "                        \"J\"\n",
            "                    ],\n",
            "                    \"last\": \"Smith\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"L\",\n",
            "                    \"middle\": [\n",
            "                        \"A\"\n",
            "                    ],\n",
            "                    \"last\": \"Morrison\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"D\",\n",
            "                    \"middle\": [\n",
            "                        \"A\"\n",
            "                    ],\n",
            "                    \"last\": \"Leib\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2002,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"76\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"2054--61\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF26\": {\n",
            "            \"ref_id\": \"b26\",\n",
            "            \"title\": \"Herpes simplex virus virion host shutoff protein: immune evasion mediated by a viral RNase?\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [\n",
            "                        \"R\"\n",
            "                    ],\n",
            "                    \"last\": \"Smiley\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2004,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"78\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"1063--1071\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF27\": {\n",
            "            \"ref_id\": \"b27\",\n",
            "            \"title\": \"The UL41 protein of herpes simplex virus 1 degrades RNA by endonucleolytic cleavage in absence of other cellular or viral proteins\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"B\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Taddeo\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"W\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Zhang\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"B\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Roizman\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2006,\n",
            "            \"venue\": \"Proc Natl Acad Sci\",\n",
            "            \"volume\": \"\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF28\": {\n",
            "            \"ref_id\": \"b28\",\n",
            "            \"title\": \"The herpes simplex virus vhs protein induces endoribonucleolytic cleavage of target RNAs in cell extracts\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"M\"\n",
            "                    ],\n",
            "                    \"last\": \"Elgadi\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"C\",\n",
            "                    \"middle\": [\n",
            "                        \"E\"\n",
            "                    ],\n",
            "                    \"last\": \"Hayes\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [\n",
            "                        \"R\"\n",
            "                    ],\n",
            "                    \"last\": \"Smiley\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 1999,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"73\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"7153--64\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF29\": {\n",
            "            \"ref_id\": \"b29\",\n",
            "            \"title\": \"Herpes simplex virus-infected cells contain a function(s) that destabilizes both host and viral mRNAs\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"A\",\n",
            "                    \"middle\": [\n",
            "                        \"D\"\n",
            "                    ],\n",
            "                    \"last\": \"Kwong\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"N\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Frenkel\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 1987,\n",
            "            \"venue\": \"Proc Natl Acad Sci U S A\",\n",
            "            \"volume\": \"84\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"1926--1956\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF30\": {\n",
            "            \"ref_id\": \"b30\",\n",
            "            \"title\": \"Direct ribosomal binding by a cellular inhibitor of translation\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"D\",\n",
            "                    \"middle\": [\n",
            "                        \"A\"\n",
            "                    ],\n",
            "                    \"last\": \"Colon-Ramos\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2006,\n",
            "            \"venue\": \"Nat Struct Mol Biol\",\n",
            "            \"volume\": \"13\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"103--114\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF31\": {\n",
            "            \"ref_id\": \"b31\",\n",
            "            \"title\": \"New insights into internal ribosome entry site elements relevant for viral gene expression\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"E\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Martinez-Salas\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"A\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Pacheco\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"P\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Serrano\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"N\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Fernandez\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2008,\n",
            "            \"venue\": \"J Gen Virol\",\n",
            "            \"volume\": \"89\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"611--637\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF32\": {\n",
            "            \"ref_id\": \"b32\",\n",
            "            \"title\": \"Endonucleolytic cleavage of eukaryotic mRNAs with stalls in translation elongation\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"K\"\n",
            "                    ],\n",
            "                    \"last\": \"Doma\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Parker\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2006,\n",
            "            \"venue\": \"Nature\",\n",
            "            \"volume\": \"440\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"561--565\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF33\": {\n",
            "            \"ref_id\": \"b33\",\n",
            "            \"title\": \"Endonuclease-mediated mRNA decay involves the selective targeting of PMR1 to polyribosome-bound substrate mRNA\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"F\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Yang\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"D\",\n",
            "                    \"middle\": [\n",
            "                        \"R\"\n",
            "                    ],\n",
            "                    \"last\": \"Schoenberg\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2004,\n",
            "            \"venue\": \"Mol Cell\",\n",
            "            \"volume\": \"14\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"435--480\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF34\": {\n",
            "            \"ref_id\": \"b34\",\n",
            "            \"title\": \"RNA quality control in eukaryotes\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"K\"\n",
            "                    ],\n",
            "                    \"last\": \"Doma\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Parker\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2007,\n",
            "            \"venue\": \"Cell\",\n",
            "            \"volume\": \"131\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"660--668\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF35\": {\n",
            "            \"ref_id\": \"b35\",\n",
            "            \"title\": \"The highways and byways of mRNA decay\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"N\",\n",
            "                    \"middle\": [\n",
            "                        \"L\"\n",
            "                    ],\n",
            "                    \"last\": \"Garneau\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Wilusz\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"C\",\n",
            "                    \"middle\": [\n",
            "                        \"J\"\n",
            "                    ],\n",
            "                    \"last\": \"Wilusz\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2007,\n",
            "            \"venue\": \"NatRev Mol Cell Biol\",\n",
            "            \"volume\": \"8\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"113--139\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF36\": {\n",
            "            \"ref_id\": \"b36\",\n",
            "            \"title\": \"Nidovirales: evolving the largest RNA virus genome\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"A\",\n",
            "                    \"middle\": [\n",
            "                        \"E\"\n",
            "                    ],\n",
            "                    \"last\": \"Gorbalenya\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"L\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Enjuanes\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Ziebuhr\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"E\",\n",
            "                    \"middle\": [\n",
            "                        \"J\"\n",
            "                    ],\n",
            "                    \"last\": \"Snijder\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2006,\n",
            "            \"venue\": \"Virus Res\",\n",
            "            \"volume\": \"117\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"17--37\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF37\": {\n",
            "            \"ref_id\": \"b37\",\n",
            "            \"title\": \"Core protein-coding sequence, but not core protein, modulates the efficiency of cap-independent translation directed by the internal ribosome entry site of hepatitis C virus\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"T\",\n",
            "                    \"middle\": [\n",
            "                        \"H\"\n",
            "                    ],\n",
            "                    \"last\": \"Wang\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [\n",
            "                        \"C\"\n",
            "                    ],\n",
            "                    \"last\": \"Rijnbrand\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"S\",\n",
            "                    \"middle\": [\n",
            "                        \"M\"\n",
            "                    ],\n",
            "                    \"last\": \"Lemon\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2000,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"74\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"11347--58\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF38\": {\n",
            "            \"ref_id\": \"b38\",\n",
            "            \"title\": \"Rift valley fever virus nonstructural protein NSs promotes viral RNA replication and transcription in a minigenome system\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"T\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Ikegami\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"C\",\n",
            "                    \"middle\": [\n",
            "                        \"J\"\n",
            "                    ],\n",
            "                    \"last\": \"Peters\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"S\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Makino\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2005,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"79\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"5606--5621\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF39\": {\n",
            "            \"ref_id\": \"b39\",\n",
            "            \"title\": \"Naturally occurring dicistronic cricket paralysis virus RNA is regulated by two internal ribosome entry sites\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"J\",\n",
            "                    \"middle\": [\n",
            "                        \"E\"\n",
            "                    ],\n",
            "                    \"last\": \"Wilson\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"J\"\n",
            "                    ],\n",
            "                    \"last\": \"Powell\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"S\",\n",
            "                    \"middle\": [\n",
            "                        \"E\"\n",
            "                    ],\n",
            "                    \"last\": \"Hoover\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"P\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Sarnow\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2000,\n",
            "            \"venue\": \"Mol Cell Biol\",\n",
            "            \"volume\": \"20\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"4990--4999\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF40\": {\n",
            "            \"ref_id\": \"b40\",\n",
            "            \"title\": \"RNase L-independent specific 28S rRNA cleavage in murine coronavirus-infected cells\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"S\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Banerjee\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"S\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"An\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"A\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Zhou\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"R\",\n",
            "                    \"middle\": [\n",
            "                        \"H\"\n",
            "                    ],\n",
            "                    \"last\": \"Silverman\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"S\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Makino\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2000,\n",
            "            \"venue\": \"J Virol\",\n",
            "            \"volume\": \"74\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"8793--802\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF41\": {\n",
            "            \"ref_id\": \"b41\",\n",
            "            \"title\": \"Lipoxygenase mRNA silencing in erythroid differentiation: The 3'UTR regulatory complex controls 60S ribosomal subunit joining\",\n",
            "            \"authors\": [\n",
            "                {\n",
            "                    \"first\": \"D\",\n",
            "                    \"middle\": [\n",
            "                        \"H\"\n",
            "                    ],\n",
            "                    \"last\": \"Ostareck\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"A\",\n",
            "                    \"middle\": [],\n",
            "                    \"last\": \"Ostareck-Lederer\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"I\",\n",
            "                    \"middle\": [\n",
            "                        \"N\"\n",
            "                    ],\n",
            "                    \"last\": \"Shatsky\",\n",
            "                    \"suffix\": \"\"\n",
            "                },\n",
            "                {\n",
            "                    \"first\": \"M\",\n",
            "                    \"middle\": [\n",
            "                        \"W\"\n",
            "                    ],\n",
            "                    \"last\": \"Hentze\",\n",
            "                    \"suffix\": \"\"\n",
            "                }\n",
            "            ],\n",
            "            \"year\": 2001,\n",
            "            \"venue\": \"Cell\",\n",
            "            \"volume\": \"104\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"281--90\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF42\": {\n",
            "            \"ref_id\": \"b42\",\n",
            "            \"title\": \"RNA 1 and RNA 2, used for d. (b and c) After incubation of RRL with GST, nsp1 (wt), or nsp1-mt (mt) for 10 min at 4\\u00b0C, Ren-HCV-FF RNA (b) or Ren-CrPV-FF RNA (c) was added to each sample and incubated at 30\\u00b0C for 25 min. Then, RNAs were extracted and subjected to Northern blot analysis using a digoxigenin-labeled probe that binds to fLuc ORF. (b) RNA, the untreated Ren-HCV-FF RNA. (c) fLuc, the RNA encoding fLuc gene; RNA, the untreated Ren-CrPV-FF RNA. (d) RRL was incubated with GST, nsp1 (wt), or nsp1-mt (mt) for 10 min at 4\\u00b0C. Ren-EMCV-FF RNA was added to each sample and incubated at 30\\u00b0C for 10 min. RNAs were extracted and subjected to Northern blot analysis using a probe that binds to rLuc ORF (rLuc probe) or fLuc ORF (fLuc probe)\",\n",
            "            \"authors\": [],\n",
            "            \"year\": null,\n",
            "            \"venue\": \"Nsp1 induces RNA cleavage at the IRES in Ren-EMCV-FF RNA (a) Schematic diagram of Ren-EMCV-FF RNA and control RNAs\",\n",
            "            \"volume\": \"\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"\",\n",
            "            \"other_ids\": {}\n",
            "        },\n",
            "        \"BIBREF44\": {\n",
            "            \"ref_id\": \"b44\",\n",
            "            \"title\": \"Author manuscript; available in PMC\",\n",
            "            \"authors\": [],\n",
            "            \"year\": 2010,\n",
            "            \"venue\": \"Nat Struct Mol Biol\",\n",
            "            \"volume\": \"\",\n",
            "            \"issn\": \"\",\n",
            "            \"pages\": \"\",\n",
            "            \"other_ids\": {}\n",
            "        }\n",
            "    },\n",
            "    \"ref_entries\": {\n",
            "        \"FIGREF0\": {\n",
            "            \"text\": \"Effects of nsp1 on cap-dependent translation and IRES-mediated translation in RRL(a) Increasing amounts of rLuc RNA were incubated in RRL in the presence of 1 \\u03bcg of nsp1 (wt), GST or nsp1-mt (mt) at 4\\u00b0C for 30 min; the molar ratios of rLuc RNA to nsp1 protein ranged from 1:10 to 1:2,000. Then, the samples were incubated for 30 min at 30\\u00b0C, and the rLuc activities were measured. Each data point represents the average of three independent experiments, and the standard deviation is shown.(b) A mixture of 0.25 \\u03bcg of rLuc RNA and 1 \\u03bcg of nsp1, GST or nsp1-mt was incubated in RRL in the presence of [ 35 S]methionine at 30\\u00b0C for 10, 20 and 30 min. Translated products were analyzed on SDS-PAGE and detected by autoradiography. (c) Schematic diagram of various RNA transcripts used for d and e. (d) Increasing amounts of GST, nsp1, or nsp1-mt were incubated with RRL for 10 min at 4\\u00b0C. Then, the samples were incubated at 30\\u00b0C for 30 min with 0.5 \\u03bcg of m 7 G-FF RNA, Ren-HCV-FF RNA, or Ren-CrPV-FF RNA in the presence of [ 35 S]methionine.\",\n",
            "            \"latex\": null,\n",
            "            \"type\": \"figure\"\n",
            "        },\n",
            "        \"FIGREF1\": {\n",
            "            \"text\": \"Toe printing analysis (a) and primer extension analysis (b)\",\n",
            "            \"latex\": null,\n",
            "            \"type\": \"figure\"\n",
            "        },\n",
            "        \"FIGREF2\": {\n",
            "            \"text\": \"Nsp1-induced modification of rLuc RNA in RRL(a) Cap-labeled rLuc RNA and the 3'-end-labeled rLuc RNA were incubated with GST, nsp1 (wt) or nsp1-mt (mt) in RRL for 15 min at 30\\u00b0C in the presence of CHX and GMP-PNP. RNAs were extracted, resolved on a sequencing gel and visualized by autoradiography. The asterisk represents a signal that migrated slightly faster than the full-length rLuc RNA in the sample incubated in the presence of nsp1. The 28S and 18S ribosomal RNA levels, detected by ethidium bromide staining, are shown as loading controls. (b) Experiments shown in (a) were repeated three times, and the rLuc RNA band intensities were determined by densitometric analysis. For the 3'-labeled rLuc RNA in the sample that was incubated in the presence of nsp1, a sum of the band intensities of the full-length rLuc RNA signal and a signal represented by the asterisk in (a) is shown in the graph. (c) RRL was incubated with GST, nsp1 (wt), or nsp1-mt (mt) for 10 min at 4\\u00b0C. Then, rLuc RNA was added to each sample and incubated at 30\\u00b0C for 20 min in the presence of [ 35 S] methionine. Aliquots of the samples were analyzed by SDS-PAGE (top panel). After addition of the same amount of CAT RNA to each sample, RNAs were extracted and translated using fresh RRL in the presence of [ 35 S] methionine. The samples were analyzed on a 12.5% SDS-PAGE gel. The middle and bottom panels represent rLuc protein and CAT protein, respectively. (d) Experiments described in (c) were repeated three times, and the abundances of rLuc protein in the first (1 st ) and second (2 nd ) translation reactions are shown.\",\n",
            "            \"latex\": null,\n",
            "            \"type\": \"figure\"\n",
            "        },\n",
            "        \"FIGREF3\": {\n",
            "            \"text\": \"Importance of ribosomes for nsp1's function(a) Cap-labeled rLuc RNA was incubated with GST, nsp1 (wt), or nsp1-mt (mt) in ribosome-free RRL for 15 min at 30\\u00b0C in the presence of CHX and GMP-PNP. RNAs were extracted, resolved on a sequencing gel, and visualized by autoradiography. (b) The band intensity of rLuc RNA shown in (a) was determined by densitometric analysis. The results represent the mean of three independent experiments. (c) Experiments were performed as described inFig. 5c, except that ribosome-free RRL was used for the initial translation reaction. (d) Cap-labeled rLuc RNA was incubated with GST, nsp1 (wt), or nsp1-mt (mt) in RRL in the presence of hippuristanol. RNAs were extracted, resolved on a sequencing gel and visualized by autoradiography. Struct Mol Biol. Author manuscript; available in PMC 2010 May 01.\",\n",
            "            \"latex\": null,\n",
            "            \"type\": \"figure\"\n",
            "        },\n",
            "        \"FIGREF4\": {\n",
            "            \"text\": \"Nsp1 inactivates the translational function of 40S ribosomal subunitRRL was incubated with GST, nsp1 (wt), or nsp1-mt (mt) for 10 min at 4\\u00b0C. Ren-CrPV-FF RNA was added to each sample and incubated at 30\\u00b0C for 25 min in the presence of [ 35 S] methionine. Aliquots of the samples were used for SDS-PAGE (a, 1 st ). RNAs were extracted from the remaining samples after addition of the same amount of CAT RNA to each sample.\",\n",
            "            \"latex\": null,\n",
            "            \"type\": \"figure\"\n",
            "        },\n",
            "        \"TABREF0\": {\n",
            "            \"text\": \"et al. Page 16 Nat Struct Mol Biol. Author manuscript; available in PMC 2010 May 01.\",\n",
            "            \"latex\": null,\n",
            "            \"type\": \"table\"\n",
            "        },\n",
            "        \"TABREF1\": {\n",
            "            \"text\": \"Nat Struct Mol Biol. Author manuscript; available in PMC 2010 May 01.\",\n",
            "            \"latex\": null,\n",
            "            \"type\": \"table\"\n",
            "        }\n",
            "    },\n",
            "    \"back_matter\": [\n",
            "        {\n",
            "            \"text\": \"We thank Peter Sarnow (Stanford University, CA) for the pGL3/Ren/CrPV /FF plasmid, Stanley Lemon (University of Texas Medical Branch at Galveston) for the pRC22F plasmid and Jerry Pelletier (McGill University, Canada) and Junichi Tanaka (University of the Ryukyus, Japan) for hippuristanol. This work was supported by\",\n",
            "            \"cite_spans\": [],\n",
            "            \"ref_spans\": [],\n",
            "            \"section\": \"Acknowledgments\"\n",
            "        }\n",
            "    ]\n",
            "}\n"
          ],
          "name": "stdout"
        }
      ]
    }
  ]
}